src/pybind/mgr/cli_api/module.py

```diff
@@ -0,0 +1,89 @@
+import json
+import logging
+import threading 
+import time
+from functools import partial
+from threading import Event, Lock
+from queue import Queue
+
+from mgr_module import CLICommand, HandleCommandResult, MgrModule
+
+logger = logging.getLogger()
+
+class CLI(MgrModule):
+
+    NATIVE_OPTIONS = [
+        'mgr_tick_period',
+    ]
+
+    @CLICommand('mgr api get')
+    def api_get(self, arg: str):
+        '''
+        Called by the plugin to fetch named cluster-wide objects from ceph-mgr.
+        :param str data_name: Valid things to fetch is osd_map
```

Comment:
I undertsan from the comment that only `osd_map` is allowed but I was able to retrieve the `service_map`:
```
[root@ceph2 ceph]# ceph mgr api get service_map
2021-06-17T11:52:43.333+0000 7fe44c88c700 -1 WARNING: all dangerous and experimental features are enabled.
2021-06-17T11:52:43.345+0000 7fe44c88c700 -1 WARNING: all dangerous and experimental features are enabled.
{"epoch": 6, "services": {"rgw": {"daemons": {"summary": "", "4402": {"start_epoch": 2, "gid": 4402, "addr": "172.20.0.2:0/2769796477", "metadata": {"arch": "x86_64", "ceph_release": "quincy", "ceph_version": "ceph version 17.0.0-5283-g3c870731 (3c870731a740807244afd63ae233e9f5a542a89e) quincy (dev)", "ceph_version_short": "17.0.0-5283-g3c870731", "cpu": "Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz", "distro": "centos", "distro_description": "CentOS Linux 8", "distro_version": "8", "frontend_config#0": "beast port=8000", "frontend_type#0": "beast", "hostname": "ceph2", "id": "8000", "kernel_description": "#1 SMP Thu Jun 10 14:19:48 UTC 2021", "kernel_version": "5.12.10-200.fc33.x86_64", "mem_swap_kb": "4194300", "mem_total_kb": "32236628", "num_handles": "1", "os": "Linux", "pid": "5321", "zone_id": "8864de6d-c72d-4963-ae94-fdc10b55a7e1", "zone_name": "default", "zonegroup_id": "3267d0cf-4401-4c16-bbd7-758e5544efad", "zonegroup_name": "default"}, "task_status": {}, "start_stamp": "2021-06-17T11:11:31.920586+0000"}}}}, "modified": "2021-06-17T11:50:16.507045+0000"}
3.719329833984375e-05
```